src/CloudKit/CKCompat.cs

```diff
@@ -137,6 +138,32 @@ public partial class CKDiscoveredUserInfo {
 		[iOS (9,0)][Mac (10,11)]
 		[Obsolete ("Empty stub (not public API).")]
 		public virtual CNContact DisplayContact { get; }
+
+#if NET
+		[UnsupportedOSPlatform ("ios15.0")]
+		[UnsupportedOSPlatform ("macos12.0")]
+#else
+		[Deprecated (PlatformName.MacOSX, 10, 11, message : "Use 'DisplayContact.GivenName'.")]
+		[Deprecated (PlatformName.iOS, 9, 0, message : "Use 'DisplayContact.GivenName'.")]
+		[Obsoleted (PlatformName.MacOSX, 12, 0, message : "Use 'DisplayContact.GivenName'.")]
+		[Obsoleted (PlatformName.iOS, 15, 0, message : "Use 'DisplayContact.GivenName'.")]
+#endif
+		public virtual string FirstName {
+			get { return null; }
+		}
+
+#if NET
+		[UnsupportedOSPlatform ("ios15.0")]
+		[UnsupportedOSPlatform ("macos12.0")]
+#else
+		[Deprecated (PlatformName.MacOSX, 10, 11, message : "Use 'DisplayContact.FamilyName'.")]
+		[Deprecated (PlatformName.iOS, 9, 0, message : "Use 'DisplayContact.FamilyName'.")]
+		[Obsoleted (PlatformName.MacOSX, 12, 0, message : "Use 'DisplayContact.GivenName'.")]
```

Comment:
Should be `DisplayContact.FamilyName` for each of these.

Reply:
Indeed, good catch! Sadly it's a backport :|
I'll fix it (another PR) in `main` and backport that fix back into `d16-10`

Reply:
https://github.com/xamarin/xamarin-macios/pull/11923